stable/goldilocks/templates/dashboard-deployment.yaml

```diff
@@ -74,11 +74,11 @@ spec:
               protocol: TCP
           livenessProbe:
             httpGet:
-              path: {{.Values.dashboard.basePath | default "/" }}health
+              path: {{ include "goldilocks.basePath" . }}/health
               port: http
           readinessProbe:
             httpGet:
-              path: {{.Values.dashboard.basePath | default "/" }}health
+              path: {{ include "goldilocks.basePath" . }}/health
```

Comment:
```suggestion
              path: {{ trimSuffix "/" .Values.dashboard.basePath }}/health
```

Reply:
That is way better, thanks for the suggestion!